src/test-utils/render.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Is there any way to overcome the warning that shows up when enabling this rule?
If it's helpful for you, this is how I handle it:

Add 
```
    "import/no-extraneous-dependencies": ["error", {"devDependencies": true }],
```
to `rules` section of `.eslintrc.js`

Reply:
Is there any way to overcome the warning that shows up when enabling this rule? >> As far as I know no or all least I did not found it.

Ooh using that rule with the file as target seems great, I will do the change